server/base/src/main/java/org/apache/accumulo/server/rpc/ThriftServerType.java

```diff
@@ -28,7 +28,11 @@
  * benchmarks against "unsecure" Accumulo use the same type of Thrift server.
  */
 public enum ThriftServerType {
-  CUSTOM_HS_HA("custom_hs_ha"), THREADPOOL("threadpool"), SSL("ssl"), SASL("sasl");
+  CUSTOM_HS_HA("custom_hs_ha"),
+  THREADPOOL("threadpool"),
+  SSL("ssl"),
+  SASL("sasl"),
+  THREADED_SELECTOR("threaded_selector");
```

Comment:
This is probably going to rejoin on formatting.

Reply:
I didn't change it, its just what the formatter decided to do.